pkg/handlers/update.go

```diff
@@ -239,7 +235,27 @@ func (h *Handler) GetAvailableUpdates(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	latestLicense, _, err := license.Sync(app, "", false)
+	latestLicense, err := store.GetLatestLicenseForApp(app.ID)
```

Comment:
looks like this is only used for airgap? can probably be moved inside the if statement below